unified/install.sh

```diff
@@ -88,6 +86,31 @@ if [ -z "$prefix" ]; then
     fi
 fi
 
+if [ -z "$python3" ]; then
+    if $nonroot; then
+        python3=$prefix/python3/bin/python3
+    else
+        python3=$prefix/python3/bin/python3
+    fi
+fi
+
+if [ -f "/etc/os-release" ]; then
+    . /etc/os-release
+elif [ -f "/etc/arch-release" ]; then
+    export ID=arch
+else
+    echo "/etc/os-release missing."
+    exit 1
+fi
```

Comment:
We probably don't need /etc/arch-release since we just want to detect Debian variants.
Also, We don't have to abort setup when /etc/os-release missing, we just able to apply /etc/sysconfig as default.
So I guess
```
if [ -f "/etc/os-release" ]; then
    . /etc/os-release
fi
```
is enough for this.
If there is no file, ID is empty, sysconfdir will be /etc/sysconfig.

Reply:
OK. fixed.